seba/stimulus_annotations.py

```diff
@@ -0,0 +1,18 @@
+# stimulus_annotations.py
+import numpy as np
+from pynwb import NWBFile
+from pynwb.misc import AnnotationSeries
+
+def add_cs_onsets_from_path(nwb: NWBFile, cs_path: str, label: str = "CS") -> NWBFile:
+    """
+    Load CS onset times (probe seconds) from a text file and add them as a stimulus AnnotationSeries.
+    """
+    cs_times = np.loadtxt(cs_path)  # seconds
+    cs_series = AnnotationSeries(
+        name="CS_onset",
+        data=[label] * int(len(cs_times)),
```

Comment:
`len` is always an `int` . In general this function makes no sense as it doesn't abstract anything really. Has to be made to generalize to different stimuli properly - not sure if that's doable sensibly.